examples/deno.ts

```diff
@@ -16,7 +16,7 @@ const plugin = await createPlugin(filename, {
 
 console.log('calling', { filename, funcname, input });
 const res = await plugin.call(funcname, new TextEncoder().encode(input));
-console.log(res)
+console.log(res);
```

Comment:
`just format` caught a bunch of missing semicolons, apologies. I'll call out the big changes.